draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1568,11 +1568,11 @@ algorithm documented in Sections {{MP_KEY}} and {{handshaking}}. The
 security of the MP-DCCP connection depends on the use of keys that are
```

Comment:
Is it still a hash-based handshake algorithm? Not sure if it referred to the generation of the token? Or it refers to the HMAC perhaps?

Reply:
Added reference to MP_HMAC section in https://github.com/tsvwg/ietf-multipath-dccp/pull/269/commits/24f9bcba139584ce3eef3dd9f628c81a7b4de758